pkgs/games/cataclysm-dda/bn-git.nix

```diff
@@ -0,0 +1,63 @@
+{ lib, callPackage, fetchFromGitHub, fetchpatch, pkgs, wrapCDDA, attachPkgs
+, tiles ? true
+, debug ? false
+, useXdgDir ? false
+, version ? "2025-05-14"
+, rev ? "5fe71bf02f544436e0f218ed53f4a4f0f560be6b"
+, sha256 ? "sha256-hfeHrvZjo7ttLG4SQmlJ//xUt8wUU9DY1K9FrVpZKGA="
+}:
+
+let
+  desktopFilePath = "";
+
+  common = callPackage ./common.nix {
+    inherit tiles debug useXdgDir desktopFilePath;
+  };
+
+  self = common.overrideAttrs (common: rec {
+    pname = "cataclysm-bn-git";
+    inherit version;
```

Comment:
usual naming scheme would be "0-unstable-YYYY-MM-DD", so `version = "0-unstable-${version}";` here

Reply:
Or rather, `(last stable version)-unstable-...`

Reply:
fixed